pages/advanced-algorithms/available-algorithms/algo.mdx

```diff
@@ -0,0 +1,442 @@
+```plaintext
+{"nodes":[{"id":18,"labels":["Node1"],"properties":{},"type":"node"},{"id":19,"labels":["Node2"],"properties":{},"type":"node"},{"id":20,"labels":["Node3"],"properties":{},"type":"node"}],"relationships":[{"id":32,"start":19,"end":18,"label":"CONNECTED","properties":{},"type":"relationship"},{"id":33,"start":20,"end":19,"label":"CONNECTED","properties":{},"type":"relationship"}],"type":"path"},
+{"nodes":[{"id":18,"labels":["Node1"],"properties":{},"type":"node"},{"id":19,"labels":["Node2"],"properties":{},"type":"node"},{"id":20,"labels":["Node3"],"properties":{},"type":"node"}],"relationships":[{"id":32,"start":19,"end":18,"label":"CONNECTED","properties":{},"type":"relationship"},{"id":26,"start":19,"end":20,"label":"CONNECTED","properties":{},"type":"relationship"}],"type":"path"},
+{"nodes":[{"id":18,"labels":["Node1"],"properties":{},"type":"node"},{"id":19,"labels":["Node2"],"properties":{},"type":"node"},{"id":20,"labels":["Node3"],"properties":{},"type":"node"}],"relationships":[{"id":25,"start":18,"end":19,"label":"CONNECTED","properties":{},"type":"relationship"},{"id":33,"start":20,"end":19,"label":"CONNECTED","properties":{},"type":"relationship"}],"type":"path"},
+{"nodes":[{"id":18,"labels":["Node1"],"properties":{},"type":"node"},{"id":19,"labels":["Node2"],"properties":{},"type":"node"},{"id":20,"labels":["Node3"],"properties":{},"type":"node"}],"relationships":[{"id":25,"start":18,"end":19,"label":"CONNECTED","properties":{},"type":"relationship"},{"id":26,"start":19,"end":20,"label":"CONNECTED","properties":{},"type":"relationship"}],"type":"path"},
+{"nodes":[{"id":18,"labels":["Node1"],"properties":{},"type":"node"},{"id":20,"labels":["Node3"],"properties":{},"type":"node"}],"relationships":[{"id":30,"start":18,"end":20,"label":"CONNECTED","properties":{},"type":"relationship"}],"type":"path"}
+```
+
+### `cover()`
+
+Returns relationships between every two nodes in the list, including self relationship.
+
+{<h4> Input: </h4>}
+
+- `nodes: List[Node]` ➡ A list of all the input nodes.
+
+{<h4> Output: </h4>}
+
+- `rel: Relationship` ➡ for each relationship between nodes, a separate record is created, and a relationship is returned.
+
+{<h4> Usage: </h4>}
+
+Graph creation:
+
+```cypher
+CREATE (d:Dog)-[s:SELF_REL]->(d), (d)-[l:LOVES]->(h:Human), (h)-[li:LIVES_IN]->(ho:House);
+```
+![](/pages/advanced-algorithms/available-algorithms/algo/cover_graph.png)
+
+```cypher
+MATCH (d:Dog),(ho:House),(h:Human)
+CALL algo.cover([d, ho, h]) YIELD rel
+RETURN startNode(rel), rel, endNode(rel);
+```
+Result:
+
+![](/pages/advanced-algorithms/available-algorithms/algo/cover_result.png)
+
+As we can see from the result, `algo.cover` returned relationships between each two nodes in the input list, including self relationships.
+
+
+### `astar()`
+
+Runs the A * search algorithm between the start and target node.
+Supports either `Numeric` or `Duration` data types for distance and heuristic properties.
```

Comment:
```suggestion
Supports either `Numeric` or `Duration` data types for distance and heuristic properties.

The default heuristic works with the assumption that the
nodes in the graph represent geospatial points and that they have defined properties for latitude and longitude.
In that case, the heuristic for each node is [haversine distance](https://en.wikipedia.org/wiki/Haversine_formula)
between the current node and the target node on Earth, and is returned in kilometers. 

In case you don't want to use geospatial types, or you want a custom heuristic, check the supported [configuration](#configuration).
```